tests/src/main/java/net/neoforged/neoforge/debug/capabilities/ItemEnergyTests.java

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright (c) NeoForged and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.debug.capabilities;
-
-import com.mojang.serialization.Codec;
-import java.util.function.Supplier;
-import net.minecraft.core.component.DataComponentType;
-import net.minecraft.core.registries.Registries;
-import net.minecraft.network.codec.ByteBufCodecs;
-import net.minecraft.world.item.Item;
-import net.minecraft.world.item.ItemStack;
-import net.neoforged.neoforge.capabilities.Capabilities.EnergyStorage;
-import net.neoforged.neoforge.capabilities.RegisterCapabilitiesEvent;
-import net.neoforged.neoforge.energy.ComponentEnergyStorage;
-import net.neoforged.neoforge.energy.IEnergyStorage;
-import net.neoforged.neoforge.registries.DeferredItem;
-import net.neoforged.neoforge.registries.DeferredRegister;
-import net.neoforged.testframework.DynamicTest;
-import net.neoforged.testframework.TestFramework;
-import net.neoforged.testframework.annotation.ForEachTest;
-import net.neoforged.testframework.annotation.OnInit;
-import net.neoforged.testframework.annotation.TestHolder;
-import net.neoforged.testframework.gametest.EmptyTemplate;
-import net.neoforged.testframework.gametest.GameTest;
-import net.neoforged.testframework.registration.DeferredItems;
-import net.neoforged.testframework.registration.RegistrationHelper;
-
-@ForEachTest(groups = "capabilities.itemenergy")
-public class ItemEnergyTests {
```

Comment:
Don't remove these old tests. Keep them, and just wrap the new handler as an old `IEnergyStorage`.